myTrip/comment/views.py

```diff
@@ -5,32 +5,68 @@
 from django.views.generic.base import View
 
 from .models import Comment
+from registration.models import CustomUser
+from trip.models import Trip
+from checkpoint.models import Checkpoint
+from photo.models import Photo
 
 
 class CommentView(View):
     """Comments view handles GET, POST, PUT, DELETE requests."""
 
-    def get(self, request, comment_id):
+    def get(self, request, comment_id=None, trip_id=None, checkpoint_id=None, photo_id=None):
         """Handles GET request.
```

Comment:
add description of what this method does
example: shows a comment if there is one, or all comments when filtering